src/lib/setup/api.ts

```diff
@@ -0,0 +1,12 @@
+import { API } from '@discordjs/core/http-only';
+import { container } from '@skyra/http-framework';
+
+export function setupAPI() {
+	container.api = new API(container.rest);
+}
```

Comment:
should this be part of HFx?

Reply:
Perhaps once the new version is released with `/http-only`, otherwise we'll load `@discordjs/ws` and a lot of unneeded bloat.